### PR TITLE
Added setLocale for Qute Templates

### PR DESCRIPTION
**Pull request by geertvanleemputten**
Added convenience method to indicate how to localize qute templates in cases where this needs to be done manually.

eg: mail templates

### CODE REVIEW

**Review by gastaldi — CHANGES_REQUESTED** [review]
Have you built the code locally? It looks incorrect to me

**Comment by geertvanleemputten** [comment]
I was unable to build due to:
Could not find artifact io.quarkus:quarkus-enforcer-rules:jar:999-SNAPSHOT

Included your changes now.

**Comment by gastaldi** [comment]
Looks good. Can you please squash all your commits into a single one? Thanks!

**Comment by geertvanleemputten** [comment]
squashed

**Comment by gastaldi** [comment]
There are conflicts. Can you make sure your branch is rebased onto the latest `main`?

File: independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateInstance.java [hunk 1]
```diff
@@ -53,6 +53,24 @@ default TemplateInstance data(String key, Object data) {
...
+        return setLocale(Locale.forLanguageTag(locale));
+    }
+
+    /**
+     * Sets the locale.
+     * @param locale
+     * @return
+     */
+    default TemplateInstance setLocale(Locale locale) {
+        return setAttribute("locale", locale);
```
Comment (gastaldi) [inline independent-projects/qute/core/src/main/java/io/quarkus/qute/TemplateInstance.java:71]:
```suggestion
        return setAttribute(MessageBundles.ATTRIBUTE_LOCALE, locale);
```

**Pull request closed without merging**